RxCocoa/RxCocoa/iOS/UITextField+Rx.swift

```diff
@@ -12,21 +12,31 @@ import UIKit
 
 extension UITextField {
     public func rx_text() -> Observable<String> {
+        return observableForControlEvent(.EditingChanged)
+    }
+
+    public func rx_endOnExit() -> Observable<String> {
```

Comment:
Would it maybe make more sense to name it rx_editingDidEndOnExit?

When I was reading the name it was kind of hard to understand. What do you think?